### PR TITLE
Add a token transition overview (SOFTWARE-5076)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: docs/release/updating-to-osg-36.md [hunk 1]
```diff
@@ -1,26 +1,59 @@
...
-    Due to potentially disruptive changes in protocols, contact your VO(s) to verify that they support token-based
-    authentication and/or HTTP-based data transfer before considering an upgrade to OSG 3.6.
-    If your VO(s) don't support these new protocols or you don't know which protocols your VO(s) support,
-    install or remain on the [OSG 3.5 release series](notes.md).
+    Due to potentially disruptive changes in protocols, consult the
+    [collaboration support tables](../security/tokens/overview.md#collaboration-support) to verify that your
+    collaboration(s) support token-based authentication and/or WebDAV/XRootD-based data transfer before considering an
+    upgrade to OSG 3.6.
+    If your collaboration(s) don't support these new protocols, install or remain on the
+    [OSG 3.5 release series](notes.md).
```
Comment (matyasselmeci) [inline docs/release/updating-to-osg-36.md:7]:
```suggestion
    [OSG 3.5 release series, with the osg-upcoming repositories enabled](notes.md).
```
or something. We shouldn't tell people to use 3.5 without upcoming anymore.

File: docs/release/updating-to-osg-36.md [hunk 2]
```diff
@@ -69,15 +102,16 @@ Updating Your OSG Compute Entrypoint
...
 !!! danger "Before considering an upgrade to OSG 3.6&hellip;"
-    Due to potentially disruptive changes in authentication methods, contact your VO(s) to verify that they support
-    token-based authentication before considering an upgrade to OSG 3.6.
-    If your VO(s) don't support these new authentication methods or you don't know which authentication methods your
-    VO(s) support, install or remain on the [OSG 3.5 release series](notes.md)
+    Due to potentially disruptive changes in protocols, consult the
+    [collaboration support tables](../security/tokens/overview.md#pilot-job-submission) to verify that your
+    collaboration(s) support token-based authentication before considering an upgrade to OSG 3.6.
+    If your collaboration(s) don't support these new authentication methods, install or remain on the
+    [OSG 3.5 release series](notes.md)
```
Comment (matyasselmeci) [inline docs/release/updating-to-osg-36.md:109]: same comment about upcoming.

**Review by matyasselmeci — APPROVED** [review]
Pre-approving; see my comments though.

File: docs/release/updating-to-osg-36.md (hunk 2 above)
Comment (brianhlin) [reply docs/release/updating-to-osg-36.md]: Done

**Pull request merged**